src/common/depthmap_toolkit/depthmap.py

```diff
@@ -88,76 +89,164 @@ def process(dir_path: str, depth: str, rgb: str):
     return width, height, depth_scale, max_confidence, data, matrix
 
 
-def get_angle_between_camera_and_floor(width: int, height: int, calibration: List[List[float]], matrix: list):
+def get_angle_between_camera_and_floor(width: int, height: int, calibration: List[List[float]], matrix: list) -> float:
+    """Calculate an angle between camera and floor based on device pose"""
     centerx = float(width / 2)
     centery = float(height / 2)
     vector = utils.convert_2d_to_3d_oriented(calibration[1], centerx, centery, 1.0, width, height, matrix)
     angle = 90 + math.degrees(math.atan2(vector[0], vector[1]))
     return angle
 
 
-def show_result(width: int, height: int, calibration: List[List[float]],
-                data: bytes, depth_scale: float, max_confidence: float, matrix: list):
+def get_floor_level(width: int,
+                    height: int,
+                    calibration: List[List[float]],
+                    data: bytes,
+                    depth_scale: float,
+                    max_confidence: float,
+                    matrix: list) -> float:
+    """Calculate an altitude of the floor in the world coordinates"""
+    altitudes = []
+    for x in range(width):
+        for y in range(height):
+            normal = utils.calculate_normal_vector(calibration[1], x, y, width, height, data, depth_scale, matrix)
+            if abs(normal[1]) > 0.5:
+                depth = utils.parse_depth(x, y, width, height, data, depth_scale)
+                point = utils.convert_2d_to_3d_oriented(calibration[1], x, y, depth, width, height, matrix)
+                altitudes.append(point[1])
+    return statistics.median(altitudes)
+
+
+def detect_child(output: object,
+                 x: int,
+                 y: int,
+                 width: int,
+                 height: int,
+                 calibration: List[List[float]],
+                 data: bytes,
+                 depth_scale: float,
+                 max_confidence: float,
+                 matrix: list,
+                 floor: float) -> float:
+
+    # highlight the focused child/object using seed algorithm
+    highest = floor
+    dirs = [[-1, 0], [1, 0], [0, -1], [0, 1]]
+    pixel = [int(width / 2), int(height / 2)]
+    stack = [pixel]
+    while (len(stack) > 0):
+
+        # get a next pixel from the stack
+        pixel = stack.pop()
+        depth_center = utils.parse_depth(pixel[0], pixel[1], width, height, data, depth_scale)
+
+        # add neighbor points (if there is no floor and they are connected)
+        if output[pixel[0]][SUBPLOT_SEGMENTATION * height + height - pixel[1] - 1][2] < 0.1:
```

Comment:
```suggestion
        index = SUBPLOT_SEGMENTATION * height + height - pixel[1] - 1
        if output[pixel[0]][index][2] < 0.1:
```

This index can be used to shorten below lines. 
Maybe we find a better name then `index`